src/gui/gui.cpp

```diff
@@ -47,6 +47,22 @@ static GLfloat screenSize[2];
 static void Gui_FillCrosshairBuffer();
 static void Gui_FillBackgroundBuffer();
 
+static void Gui_DrawAmmunitionCount()
+{
+    char ammunitionsCountString[11]; // Should have room enough to store a 10-digit number and the string terminating zero
+    int ammunitionInventoryItemId, ammunitionCount;
+
+    // Retrieve currently used ammunition
+    entity_p player = World_GetPlayer();
+    ammunitionInventoryItemId = player->character->currentAmmunitionInventoryItemId;
+    if((ammunitionInventoryItemId == ITEM_PISTOL_AMMO) || (ammunitionInventoryItemId == ITEM_GRAPPLEGUN_AMMO)) return; // Do not display ammunition amount for unlimited ammunition weapons
+
+    // Display ammunition count to top right screen corner
+    ammunitionCount = Inventory_GetItemsCount(player->inventory, ammunitionInventoryItemId);
+    sprintf(ammunitionsCountString, "%d", ammunitionCount);
```

Comment:
snprintf - less compiler warnings

Reply:
Ok, I will change this in another commit.